include/swift/AST/Attr.def

```diff
@@ -510,19 +510,19 @@ DECL_ATTR(differentiable, Differentiable,
   90)
 DECL_ATTR(differentiating, Differentiating,
   OnFunc | LongAttribute | AllowMultipleAttributes |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove |
+  ABIStableToAdd | ABIBreakingToRemove | APIStableToAdd | APIBreakingToRemove |
   NotSerialized, 91)
 SIMPLE_DECL_ATTR(compilerEvaluable, CompilerEvaluable,
   OnAccessor | OnFunc | OnConstructor | OnSubscript |
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove |
   NotSerialized, 92)
 SIMPLE_DECL_ATTR(noDerivative, NoDerivative,
   OnVar |
-  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd | APIBreakingToRemove,
```

Comment:
Note: I made the stylistic choice to intentionally leave this line >80 columns so all the API/ABI status can be on one line.